### PR TITLE
 (L10n) Delivering translations for POT and README files

### PR DESCRIPTION
**Pull request by ehom**

### CODE REVIEW

File: readmes/README_ja_JP.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+
+# yumrepo_core
+
+#### 目次
+
+1. [説明](#description)
+2. [セットアップ - yumrepo_coreモジュール導入の基本](#setup)
+    * [yumrepo_coreモジュールの利用方法](#beginning-with-yumrepo_core)
+3. [使用 - 設定オプションと追加機能](#usage)
+4. [開発 - モジュール貢献についてのガイド](#development)
```
Comment (melissa) [inline readmes/README_ja_JP.md]: Do these links work? I would think `(#development)` should correspond to `開発`? (I have no idea if that's the correct section, I'm just guessing)

File: locales/ja/puppetlabs-yumrepo_core.po [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2018 Puppet, Inc.
+# This file is distributed under the same license as the puppetlabs-yumrepo_core package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2018.
```
Comment (melissa) [inline locales/ja/puppetlabs-yumrepo_core.po]: Where do these details come from? Should we change them to be something more realistic?

File: readmes/README_ja_JP.md (hunk 1 above)
Comment (joshcooper) [reply readmes/README_ja_JP.md]: The links don't seem to work. @ehom should that be?

```
[開発 - モジュール貢献についてのガイド](#開発)
```

Comment (ehom) [reply readmes/README_ja_JP.md]: i looked into this issue a little more. it turns out that all links from table of contents are broken in all localized markdown files. If the section header is localized, it breaks the anchor reference. Transifex doesn't expose the anchor for localization. So, to resolve this, we'll need a post-localization step to resolve this. Will create an ticket to list all broken markdown files.

Comment (ehom) [reply readmes/README_ja_JP.md]: Hi @melissa i'm waiting for a response from Transifex. Give it a day.

**Comment by melissa** [comment]
Closed in favor of https://github.com/puppetlabs/puppetlabs-yumrepo_core/pull/9

**Pull request closed without merging**